app/controllers/user_controller.rb

```diff
@@ -0,0 +1,51 @@
+ServiceDiscovery::App.controllers :users do
+  
+  post '/:user_id/address' do
+    get_create_address_params
+    address = create_address @input, params[:user_id]
+    status 201
+    {address_id: address.id}.to_json
+  end
+
+  get '/:user_id/address' do
+    addresses = Address.where('user_id' => params[:user_id])
```

Comment:
Symbolize the hash key. Have is as 
addresses = Address.where(user_id: params[:user_id])